packages/stencil/components/form-field/src/components/osds-form-field/osds-form-field.scss

```diff
@@ -0,0 +1,35 @@
+@import 'styles/osds-form-field.mixins';
+@import 'styles/osds-form-field.color';
+@import 'styles/osds-form-field.size';
+@import 'styles/osds-form-field.typography';
+@import '~@ovhcloud/ods-theming/color/ods-theming-color';
+@import '~@ovhcloud/ods-theming/ods-theme';
+
+// CSS targeted only for this component
+// /!\ for theming purposes, it has to be done in theming files
+// (i.e. packages/libraries/theming/...)
+:host {
+  width: fit-content;
```

Comment:
?

Reply:
If no width specified, input takes 100% width rather than its normal width

Reply:
Well it is in fact the behavior we will want, as we put the flex display as default now.
Probably more a concern for https://github.com/ovh/design-system/pull/89 though

Reply:
so shall we put flex as default and remove the flex attribute?